include/llbuild/BuildSystem/BuildValue.h

```diff
@@ -68,6 +68,9 @@ class BuildValue {
     /// A value produced by a command which was cancelled.
     CancelledCommand,
 
+    /// A value produced by a command which was skipped.
+    SkippedCommand,
+
```

Comment:
This commit should also technically bump `BuildSystemImpl::internalSchemaVersion`, for the same reason as before, although I'm fine just bumping once for the PR.